app/src/main/java/com/recommend_tour/data/TourPagingSource.kt

```diff
@@ -0,0 +1,42 @@
+package com.recommend_tour.data
+
+import android.util.Log
+import androidx.paging.PagingSource
+import androidx.paging.PagingState
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.withContext
+
+class TourPagingSource (
+    private val tourRepository: TourRepository,
+    private val areaName: String,
+    private val contentTypeApiCode: String
+): PagingSource<Int, TourItem>(){
```

Comment:
```suggestion
class TourPagingSource (
    private val tourRepository: TourRepository
): PagingSource<Int, TourItem>(){
fun changeAreaName(~)
fun changeCode(~)
```
현재 pagingSource 객체를 매번 새로 생성하는데
이런식으로 관련 query값들만 수정하는게 좋을듯합니다.